README.md

```diff
@@ -227,6 +227,8 @@ to this swarm. When this is set, docker will be installed on all nodes of the cl
 recommended that the swarm manager is specified on a worker node as it runs docker containers. Check
 out [Portainer] if you want to run a management UI for your swarm cluster.
 
+7. `elkserver` - Sets up the Elasticsearch, Logstash, and Kibana stack. This allows logging data to be search, analyzed, and visualized in real time. 
```

Comment:
Since there seems to be a hard dependency on Docker for the ELK stack implementation to work, I highly recommend adding a note here stating that when `elkserver` is specified, also need to specify `swarmmanager` as well.

Reply:
I do not think there's a dependency for docker at all for this. The issue with docker was originally presented because while testing my changes, I would have to run setup multiple times. When you did this (running setup repeatedly), an error occurred with docker.  I believe I tested running the `elkserver` service without `swarmmanger` on previously and I didn't have an issue.